daily.sh

```diff
@@ -136,8 +136,8 @@ check_dependencies() {
 
     python3=$(python3 -c "import sys;print(int(sys.version_info < (3, 4)))" 2> /dev/null)
     if [ "$python3" -eq 0 ]; then
-        python3 scripts/check_requirements.py > /dev/null 2>&1 || $(type -p python3) -m pip install -r requirements.txt > /dev/null 2>&1
```

Comment:
Do not change daily.sh  I'm telling you it will break installs.
Only add your requirements check in validation.